app/assets/javascripts/initializers/initializeBaseTracking.js.erb

```diff
@@ -116,6 +116,29 @@ function trackCustomImpressions() {
         successLinks[i].addEventListener('click', function() { trackHtmlVariantSuccess(dataBody, csrfToken) });
       }
     }
+
+    // page view
+    if (ArticleElement && tokenMeta && !isBot) {
+      var randomNumber = Math.floor(Math.random() * 10); // 1 in 10; Only track 1 in 10 impressions
+      if (!checkUserLoggedIn() && randomNumber != 1) {
```

Comment:
I suggest transforming condition to smth like `if !(checkUserLoggedIn() || randomNumber == 1)`, in my opinion it's easier to understand.